src/ipyaladin/__init__.py

```diff
@@ -105,6 +105,56 @@ def _handle_custom_message(self, model, message, list_of_buffers):  # noqa: ARG0
         elif event_type == "select" and "select" in self.listener_callback:
             self.listener_callback["select"](message_content)
 
+    @property
+    def target(self):
+        from astropy.coordinates import SkyCoord
+
+        try:
+            return SkyCoord(
+                ra=self._target.split(" ")[0],
+                dec=self._target.split(" ")[1],
+                frame="icrs",
+                unit="deg",
+            )
+        except ValueError as e:
+            raise RuntimeError(
+                "If target is an object, please retry "
+                "after object coordinates are fetched"
+            ) from e
+
+    @target.setter
+    def target(self, target):
+        from astropy.coordinates import SkyCoord
+
+        if isinstance(target, str):
+            if target[0].isalpha():  # If the target is an object
+                sc = SkyCoord.from_name(target)
+                self._target = f"{sc.icrs.ra.deg} {sc.icrs.dec.deg}"
+                self.send(
+                    {
+                        "event_name": "goto_ra_dec",
+                        "ra": sc.icrs.ra.deg,
+                        "dec": sc.icrs.dec.deg,
+                    }
+                )
+            else:  # If the target is a coordinate string
```

Comment:
Maybe we could just add a basic test to check if there is at least 2 values separated by a space before sending it to gotoObject of aladin. Aladin Lite does not move if we just give for example:
```python
aladin.target = '14.89'
```